app/views/alchemy/admin/pages/edit.html.erb

```diff
@@ -69,7 +69,7 @@
   <% if can?(:publish, @page) %>
     <div class="button_with_label">
       <%= form_tag alchemy.publish_admin_page_path(@page), id: 'publish_page_form' do %>
-        <%= button_tag class: 'icon_button', title: Alchemy.t(:explain_publishing) do %>
+        <%= button_tag class: 'icon_button', title: (Alchemy.t(:explain_publishing) unless @page.published_at), disabled: @page.published_at do %>
```

Comment:
Please do not disable the publish button. How is a user able to publish a non-public page if we disable the button? And the same note about the title vs. label applies here.